liota/entities/registered_entity.py

```diff
@@ -36,6 +36,16 @@ def __init__(self, ref_entity, ref_dcc, reg_entity_id):
         self.ref_entity = ref_entity
         self.ref_dcc = ref_dcc
         self.reg_entity_id = reg_entity_id
+        self.parent = None
```

Comment:
No, I do need to locally hold the relationship. Previously, we do not define it in RegisteredEntity since Entity has it. We can trace there.

When set properties, in RegisteredMetric case, reg_entity_obj.parent is used. 
Can you remember that previously AWS DCC checkin's want to let Metrics have attributes?
We can make it as properties. So it is needed.